Documentation/Signoffs/Resistance_System_Control.md

```diff
@@ -0,0 +1,218 @@
+
+#### Actuator Duty Cycle
+
+The duty cycle of the utilized linear actuator is 25 %. This value must not be exeeded during normal operation. As a result, the controller must be programmable so that the duty cycle may be limited.
+
+#### Actuator Driving Voltage
+
+The utilized actuator is rated for operation at 12 VDC. The control circuit must be capable of providing this voltage to the actuator.
+
+#### Current Capability
+
+The utilized actuator draws a stall current of 7 A, and has a nominal current draw from 1.1 A to 3.4 A. The control circuit must be capable of handling a motor stall without undergoing thermal overload. In addition, the utilized power connector must be capable of handling the total current demand.
+
+#### Wireless Communications
+
+The control ciruit must be capable of recieving a wirelessly-transmitted digital signal from the ride replay system using Bluetooth to update the actuator's position. Since human reaction time is limited to between 100 and 200 ms [1], it follows that the period of data transmission for each packet containing a position value be constrained to be less than 100 ms.
+
+#### Back-EMF Protection
+
+In the course of operation, particularly when the motor is switched off suddenly, a transient voltage potential known as back-EMF will be generated by the motor with opposite polarity to its driving voltage. Due to this, the control circuit shall contain freewheeling diodes to shunt the resulting current away from the switching transistors to prevent damage to the transistors.
+
+#### Mountability
+
+The PCB layout must account for the need to mount the circuit within its housing, as such, screw holes shall be included in the PCB design.
+
+#### Stability
+
+The device must operate without becoming unstable.
+
+#### Resolution
+
+The device must be capable of providing at least 85 unique states, or lengths of extension when operating within the specified wheel speed range (from 116 RPM to 288 RPM).
+
+### Schematic and PCB Layout
+
+![Actuator Driver Schematic](https://user-images.githubusercontent.com/118228609/216786225-c57e1eca-aaaa-4acd-9b2b-d43dd4212f03.png)
+
+Figure 1: a schematic for the control circuit.
+
+![PCB_draft4](https://user-images.githubusercontent.com/118228609/216786341-876a2da2-cf4d-43fb-b254-a66bb5e4cf97.png)
+
+Figure 2: the PCB layout for the design.
+
+The design files are included within the Electrical subdirectory of the Documentation directory. See links: [schematic](https://github.com/rjdurlin42/mariokartrev_2_team_4/blob/main/Documentation/Electrical/Ride%20Resistance%20System%20Control%20Circuit.png), [pcb](https://github.com/rjdurlin42/mariokartrev_2_team_4/tree/main/Documentation/Electrical/PCB/Sources/Ride%20Resistance%20System%20Control%20Circuit).
+
+### Analysis
+
+#### Actuator Duty Cycle
+
+Arduino microcontroller determines PWM duty cycle; operation fully programmable; therefore, constraint met.
+
+#### Actuator Driving Voltage
+
+Input voltage = 12 V
```

Comment:
This is not analysis.

Reply:
Added circuit analysis and made the constraint more realistic.